web/templates/confirm_dialog/confirm_reactivate_bot.hbs

```diff
@@ -5,3 +5,6 @@
     {{#*inline "z-user"}}<strong>{{username}}</strong>{{/inline}}
     {{/tr}}
 </p>
+{{#if original_owner_deactivated}}
+    <p>{{t "However, it will no longer be subscribed to the private streams that you are not subscribed to." }}</p>
+{{/if}}
```

Comment:
This should also explain that you'll become the owner, right? Maybe:

"Because the original owner of this bot (name) is deactivated, you will become the owner for this bot. The bot will be removed from any private streams that it was subscribed to that you do not have access to."